docs/BOOTING.md

```diff
@@ -79,9 +79,45 @@ Second stage GRUB is expected to do all the [heavy lifting](../pkg/grub/rootfs.c
+grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L143) and anyone constructing
 its content is expected to be familiar with the overall flow of read-only grub.cfg.
 
+Option to use in grub.cfg on the CONFIG partition:
+
+1. Cgroup related options. Please follow the [example](README.md#eve-cgroups) to write
+   these options in the correct variables:
+    1. `dom0_mem` - memory limit for eve cgroup (default is `750M,max:750M`)
+    2. `dom0_max_vcpus` - cpu limit for eve cgroup (default is `1`)
+    3. `eve_mem` - memory limit for cgroups with services of EVE (default is `600M,max:600M`)
+    4. `eve_max_vcpus` - cpu limit for cgroups with services of EVE (default is `1`)
+    5. `ctrd_mem` - memory limit for cgroups with containerd-shims of EVE (default is `350M,max:350M`)
+    6. `ctrd_max_vcpus` - cpu limit for cgroups with containerd-shims of EVE (default is `1`)
+2. Installer options. These options are used during [the installation](DEPLOYMENT.md) process and may be set by adding
+   line `set_global dom0_extra_args "$dom0_extra_args OPTION1=VAL1 OPTION2 "`, where OPTION fields described below
+   (`DISK` below is e.g. `sda`):
+    1. `eve_blackbox` - if set, installer will collect [information](../pkg/mkimage-raw-efi/install#L93)
+       from EVE to INVENTORY partition on installation media (it must be writable, i.e. USB flash drive)
+       and exit without installation.
+    2. `eve_nuke_disks=DISK` - clean partition table on the defined disks before install.
+       You can use multiple disks separated by comma. Please use this option if you know that other
+       disks contain old EVE installations to clean them out.
+    3. `eve_nuke_all_disks` - clean partition table on all disks. Use it carefully, it may clean installation media
+       if it is writable.
+    4. `eve_install_disk=DISK` - set particular disk to install EVE. Default behavior is to use any free disk found.
+    5. `eve_persist_disk=DISK` - set particular disk to use for persistent data (volumes, logs, etc.).
+       Default behavior is to use partition on the disk selected or defined for EVE installation.
+    6. `eve_install_server=SERVER` - override `/config/server` with value defined in `SERVER`. Default behavior is to
+       use server defined during building of installer media (e.g. `zedcloud.zededa.net` for releases).
+    7. `eve_pause_before_install` - return to shell before installation and wait for `exit`.
+    8. `eve_pause_after_install` - return to shell after installation and wait for `exit`.
+    9. `eve_reboot_after_install` - reboot after installation complete, default behavior for IPXE.
+       If not set, will poweroff device after installation.
+    10. `eve_install_skip_config` - do not install config partition onto device.
+    11. `eve_install_skip_persist` - do not install persist partition onto device.
+    12. `eve_install_skip_rootfs` - do not install rootfs partition onto device.
```

Comment:
You may want to mention that these are also tweakable from the graphical GRUB menu.

Reply:
added comment about it, thank you